### ISSUE TITLE
Going from python script to gui

### ISSUE THREAD

**Issue by wholden** [kklmn/xrt#75]
Is there a "standard" way of going from a working script simulation to being able to visualize the elements of that script in the gui?

On my own I have rigged together some python code to programmatically modify the XML and then use the gui code to translate that XML into a usable python script, but I'm wondering if there is a less roundabout way that already exists.

PS Roman I enjoyed your talk at the Global XAS journal club this morning.

**Comment by kklmn** [comment]
Hi,
Many of our examples are prepared for this. Please search them for ` showIn3D` and set it True to visualize.

**Comment by yxrmz** [comment]
Hi, yes, you need to add a call to beamLine.prepare_flow() at the end of run_process, and then run Glow  with beamLine.glow() **before** run_raytracing in the main function of your script.
I can add an inverse function, XML generator from an existing script if anyone's interested.

**Comment by wholden** [comment]
Ah perfect thank you both!

I had not visited the examples in a while and hadn't noticed the showin3d option, I'll try it out. 

I'll give it a try and test it out. If it launches xrtglow then presumably I could save the XML at that point.

**Comment by leahwang1962** [comment]
For MontelMirror,py, I changed showIn3D=true, however I got the error as shown below. 
I saw in the above message "I can add an inverse function, XML generator from an existing script if anyone's interested". Like to know if it is available now.
Can I add Gaussian source in XRTQoot when add source while right click  beamline?

 File "C:\Pycharm\Projects\20220222_xrt\xrt\gui\xrtGlow\__init__.py", line 349, in makeColorsPanel
    self.paletteWidget.setSizePolicy(qt.QSizePolicy.Maximum,
TypeError: 'PySide6.QtWidgets.QWidget.setSizePolicy' called with wrong argument types:
  PySide6.QtWidgets.QWidget.setSizePolicy(Policy, Policy)
Supported signatures:
  PySide6.QtWidgets.QWidget.setSizePolicy(PySide6.QtWidgets.QSizePolicy)
  PySide6.QtWidgets.QWidget.setSizePolicy(PySide6.QtWidgets.QSizePolicy.Policy, PySide6.QtWidgets.QSizePolicy.Policy)
